app/models/customer.rb

```diff
@@ -0,0 +1,14 @@
+class Customer < ApplicationRecord
+  
+  has_many :rentals
+
+# validations
+validates :name, presence: true, uniqueness: true
+validates :address, presence: true, uniqueness: true
+validates :city, presence: true, uniqueness: true
+validates :state, presence: true, uniqueness: true
+validates :postal_code, presence: true, uniqueness: true
```

Comment:
These fields don't make sense with the uniqueness modifier, and cause your seed file to fail.